src/features/home/Tiles.jsx

```diff
@@ -162,6 +163,22 @@ export default function Tiles() {
         </Tile>
       </Grid>
 
+      <Grid
+        item
+        md={3}
+        sm={6}
+        xs={12}
+      >
+        <Tile
+          description={t("tileSettingsDescription")}
+          disabled={isConnected && hasAdb}
```

Comment:
IMHO we can leave this tile always active since it does not really depend on any device being connected.

Reply:
oh yeah my bad, didn't realize I left the checks there